### DISCUSSION TITLE
🐛 data-tags: true problem in ModelSelect2TagWidget. automate seperate words. not working

### DISCUSSION THREAD

**Discussion by neoneo40** [codingjoe/django-select2#279] [Q&A]
### Bug Description

Due to the nature of Hangul, it consists of consonants and vowels.
If you try to add a tag as '비상금', it will change to '비상ㄱㅡㅁ' or '빗ㅏㅇ금' or '비ㅅㅏㄱㅡㅁ', etc.
For some reason, I looked it up and found that data-tags is true when using ModelSelect2TagWidget, but this phenomenon only occurs in this case.
In ModelSelect2MultipleWidget, it is normally displayed as '비상금'.
I found the cause and how to solve it, but I'm not sure how to solve it. Please solve it.[](url)
Adding data-ajax--delay alleviates this slightly.

### Steps to Reproduce

 you try to add a tag as '비상금', it will change to '빗ㅏㅇ금' or '비ㅅㅏㄱㅡㅁ'

### Expected Behavior

<img width="1192" alt="스크린샷 2024-04-06 오전 6 23 06" src="https://github.com/codingjoe/django-select2/assets/6144782/3a41ea9a-6588-4f27-9c7d-812b29545d00">

**Comment by codingjoe** [comment]
Hi there 👋

I am not too familiar with Korean, or Hangul for that matter. Therefore, I'd appreciate it if you could provide a little more information.

How is the value being stored in the database? What values do you see in your browsers network logs? Please provide me with a code example or part of your implementation. Since we don't provide this part as a feature, I am sure you have written lines of code, that store new tags. I'd be curious about this part in particular.

Please let me know if you need any help.

Cheers,
Joe

**Reply by neoneo40** [reply]
@codingjoe I wrote this problem and solution. check this code. 
[https://gist.github.com/neoneo40/c6eed0990a44bf09f58aef300b1790a7](https://gist.github.com/neoneo40/c6eed0990a44bf09f58aef300b1790a7)

**Reply by codingjoe** [reply]
Huh, I might have a guess. Is this only happening for new choices?

**Reply by neoneo40** [reply]
@codingjoe This is only a problem when creating a new tag in Korean. If you look at the network tab, it sends one character at a time. Existing Korean tags created can be searched without any problems.
